.github/workflows/tag-maintainers.yml

```
@@ -0,0 +1,141 @@
+name: Tag Module Maintainers
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+permissions:
+  contents: read
+  pull-requests: write
+  id-token: write
+jobs:
+  tag-maintainers:
+    runs-on: ubuntu-latest
+    # if: github.repository_owner == 'nix-community' && github.event.pull_request.draft == false
+    if: github.repository_owner == 'nix-community'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install Nix
+        uses: cachix/install-nix-action@v31
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+          extra_nix_config: |
+            experimental-features = nix-command flakes
+      - name: Get changed files
+        id: changed-files
+        run: |
+          CHANGED_FILES=$(git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | grep '^modules/' | grep -v '^modules/\(po\|.*\/news\)/' || true)
+          echo "Changed module files:"
+          echo "$CHANGED_FILES"
+          echo "module_files<<EOF" >> $GITHUB_OUTPUT
+          echo "$CHANGED_FILES" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+      - name: Build module maintainers lookup
+        run: nix-build --show-trace -A docs.jsonModuleMaintainers
+      - name: Find maintainers for changed files
+        id: find-maintainers
+        if: steps.changed-files.outputs.module_files != ''
+        run: |
+          MODULE_MAINTAINERS=$(cat ./result)
+
+          # Create array to store maintainers to notify
+          declare -A MAINTAINERS_TO_NOTIFY
+          PR_AUTHOR="${{ github.event.pull_request.user.login }}"
+
+          # Process each changed file
+          while IFS= read -r FILE; do
+            if [[ -z "$FILE" ]]; then
+              continue
+            fi
+
+            echo "Processing file: $FILE"
+
+            # Try different path formats to find the right one that matches the JSON
+            for PATH_FORMAT in \
+              "$FILE" \
+              "/Users/khaneliman/github/home-manager/$FILE" \
+              "/$PWD/$FILE" \
+              "$PWD/$FILE"
+            do
+              echo "Trying path format: $PATH_FORMAT"
+
+              # Get maintainers for this file using the current path format
+              MAINTAINERS=$(jq -r ".[\"$PATH_FORMAT\"] | if . != null then .[].github else empty end" <<< "$MODULE_MAINTAINERS" 2>/dev/null || true)
```

Wouldn't it be possible to make jq read the file directly?

```suggestion
              MAINTAINERS=$(jq -r ".[\"$PATH_FORMAT\"] | if . != null then .[].github else empty end" ./result 2>/dev/null || true)
```